stdpopsim/slim_engine.py

```diff
@@ -660,4 +660,6 @@ def script_file_f():
         return ts
 
 
-stdpopsim.register_engine(_SLiMEngine())
+# SLiM does not currently work on Windows.
+if sys.platform != "win32":
```

Comment:
I suppose this should be `sys.platform.startswith("win")` to catch 64 bit windows too?